### PR TITLE
DOCS-4597 -- MQ v2.1.1 release

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-4597

### CODE REVIEW

File: modules/ROOT/pages/mq/mq-release-notes.adoc [hunk 1]
```diff
@@ -11,6 +11,20 @@ See Also:
...
+== 2.1.1
+
+*April 4, 2019*
+
+* Backend and user interface improvements.
+
+=== Fixed in this Release
+
+* Audit all auditable actions even if they fail or are invalid. (MQ-629)
+* Catch Exception when tries to delete a queue and It is not in SQS. (MQ-726)
```
Comment (steveCluster) [inline modules/ROOT/pages/mq/mq-release-notes.adoc]: Fixed an exception when a user tries to delete a queue multiple times

File: modules/ROOT/pages/mq/mq-release-notes.adoc [hunk 2]
```diff
@@ -11,6 +11,20 @@ See Also:
...
+* Delete environment fails when there are a lot of destinations to delete. (MQ-736)
+* Make Okta users able to create a client app. (MQ-1499)
```
Comment (steveCluster) [inline modules/ROOT/pages/mq/mq-release-notes.adoc]: Organizations that use OKTA external identity management can now create MQ clients

File: modules/ROOT/pages/mq/mq-release-notes.adoc [hunk 3]
```diff
@@ -11,6 +11,20 @@ See Also:
...
+* Queue and message exchange is not showing the name properly when the name is too long. (MQ-1487)
```
Comment (steveCluster) [inline modules/ROOT/pages/mq/mq-release-notes.adoc]: Destination names now render properly when the name is very long.

Comment (george-hoffer) [reply modules/ROOT/pages/mq/mq-release-notes.adoc]: fixed as requested

File: modules/ROOT/pages/mq/mq-release-notes.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq/mq-release-notes.adoc]: fixed as requested

File: modules/ROOT/pages/mq/mq-release-notes.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq/mq-release-notes.adoc]: fixed as requested

**Pull request merged**